src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/MonitorProjectConfigurationFilePathParams.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+{
+    internal class MonitorProjectConfigurationFilePathParams
```

Comment:
Not today because once our language server goes OOP the LanguageServerClient dll will not have a reference to the language server dll. That being said there is an opportunity there for us to build a Razor protocol dll that has these shared contracts.